leveldown.js

```diff
@@ -55,6 +54,15 @@ function SubDown (db, prefix, opts) {
   if (prefix[0] === separator) prefix = prefix.slice(1)
   if (prefix[prefix.length - 1] === separator) prefix = prefix.slice(0, -1)
 
+  var code = separator.charCodeAt(0) + 1
+  var ceiling = String.fromCharCode(code)
+
+  Buffer.from(prefix).forEach(function (byte) {
```

Comment:
Note to self: `forEach` is `Uint8Array.prototype.forEach` which is not available in IE < 11 with `buffer@4`. That's fine here: `abstract-leveldown` (and consequently current `subleveldown`) doesn't support IE < 11 either.